### DISCUSSION TITLE
Scrutinio trimestre

### DISCUSSION THREAD

**Discussion by luigiadoc** [iisgiua/giuaschool#794] [Richieste di aiuto]
Salve @trinko in quanto nella nostra scuola l'anno scolastico è diviso in Trimestre e Pentamestre vorrei sapere se la prossima settimana sarà possibile effettuare gli scrutini sul registro.

**Comment by trinko** [comment]
Certo. Se tutto è impostato correttamente non dovrebbe esserci nessun problema.
